config/default/traefik.yaml

```diff
@@ -0,0 +1,25 @@
+globalArguments:
+  - "--global.checknewversion"
+logs:
+  general:
+    format: json
```

Comment:
why json?

Reply:
because thats how loki and promtail read logs best.